pyobis/checklist/checklist.py

```diff
@@ -115,54 +128,56 @@ def redlist(
             "enddepth": enddepth,
             "geometry": geometry,
             "flags": flags,
-        },
-        "application/json; charset=utf-8",
-        **kwargs
-    )
-    return out
-
-
-def newest(
-    scientificname=None,
-    taxonid=None,
-    nodeid=None,
-    startdate=None,
-    enddate=None,
-    startdepth=None,
-    enddepth=None,
-    geometry=None,
-    flags=None,
-    **kwargs
-):
-    """
-    Generate a checklist of most recently added species.
-
-    :param scientificname: [String] Scientific name. Leave empty to include
-        all taxa.
-    :param taxonid: [String] Taxon AphiaID.
-    :param nodeid: [String] Node UUID.
-    :param startdate: [String] Start date formatted as YYYY-MM-DD.
-    :param enddate: [String] End date formatted as YYYY-MM-DD.
-    :param startdepth: [Fixnum] Start depth, in meters. Depth below sea level are treated as
-        positive numbers.
-    :param enddepth: [Fixnum] End depth, in meters. Depth below sea level are treated as
-        positive numbers.
-    :param geometry: [String] Geometry, formatted as WKT or GeoHash.
-    :param flags: [String] Comma separated list of quality flags which need to
-        be set.
-
-    :return: A dictionary
-
-    Usage::
-
-       from pyobis import checklist as ch
-       ch.newest(scientificname='Abra Alba')
-    """
-    url = obis_baseurl + "checklist/newest"
-    scientificname = handle_arrstr(scientificname)
-    out = obis_GET(
-        url,
-        {
+        }
+        out = obis_GET(
```

Comment:
While the API uses capital letter GET, that is odd in python code. Usually capital letter are used only in classes, not in functions. It is not a big deal and you don't have to change it if this is already established in the code.